pkg/kubelet/client/kubelet_client.go

```diff
@@ -73,9 +78,20 @@ func MakeTransport(config *KubeletClientConfig) (http.RoundTripper, error) {
 	}
 
 	rt := http.DefaultTransport
-	if config.Dial != nil || tlsConfig != nil {
+	dialer := config.Dial
+	if dialer == nil && config.Lookup != nil {
+		// Assuming ConnectivityService if SSHTunnel is not turned on.
+		// We will not get a dialer if connectivity service is disabled.
+		networkContext := server.NetworkContext{ConnectivityServiceName: "cluster"}
```

Comment:
It's been a while since I read the kep, but these names are well known, right?  I'm not generally a `const` fan, but it may help follow these around since I remember the KEP calling out name based contracts for usage.

Reply:
Good point.